app/services/ajax.js

```diff
@@ -62,24 +62,12 @@ export default Ember.Service.extend({
       options.contentType = options.contentType || 'application/json; charset=utf-8';
     }
     success = options.success || (function() {});
-    options.success = (data, status, xhr) => {
```

Comment:
Or, alternatively, we could pair on removing all of the ajax calls that needs it and then you could leave it removed here.